styleguide/README.md

```diff
@@ -157,6 +157,49 @@ Use your own judgment to choose between the options.
 ---
 ### Ruby
 
+### Use a consistent class layout
+
+See [Ruby Style Guide on class layout](https://github.com/bbatsov/ruby-style-guide#consistent-classes)
```

Comment:
Ah, maybe say "Based on …" then :) I don't want us to reference that styleguide as an authority.